### PR TITLE
Added my keymap for the XD75

### PR DESCRIPTION
**Pull request by RAOEUS**
The RAOEUS keymap has been added for the XD75. I've included a stenography layer using GeminiPR protocol. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xiudi/xd75/keymaps/raoeus/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+STENO_ENABLE = yes          # Additional protocols for Stenography(+1700), requires VIRTSER
+AUTO_SHIFT_ENABLE = yes	    # When you hold down a character in QWERTY, shift is automatically applied
```
Comment (drashna) [inline keyboards/xiudi/xd75/keymaps/raoeus/rules.mk:3]:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
STENO_ENABLE = yes          # Additional protocols for Stenography requires VIRTSER
AUTO_SHIFT_ENABLE = yes	    # When you hold down a character in QWERTY, shift is automatically applied
```

File: keyboards/xiudi/xd75/keymaps/raoeus/keymap.c [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+     * | RGB-   |        |        |        |        |        |        |        | HUE-   | SAT-   | BRITE- | SPEED- |        |        |        |
+     * '--------------------------------------------------------------------------------------------------------------------------------------'
+     */
+    [_BOTH] = LAYOUT_ortho_5x15(KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, _______, _______, _______, _______, RESET, DEBUG, _______, _______, _______, _______, TERM_ON, TERM_OFF, _______, _______, KC_DEL, _______, _______, _______, _______, _______, _______, _______, _______, AG_NORM, AG_SWAP, QWERTY, STENO, _______, _______, RGB_M_P, _______, _______, _______, RGB_MOD, _______, _______, _______, _______, _______, _______, RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, _______, _______, _______, RGB_RMOD, _______, _______, _______, _______, _______, _______, _______, RGB_HUD, RGB_SAD, RGB_VAD, RGB_SPD, _______, _______, _______)};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case QWERTY:
+            if (record->event.pressed) {
+                layer_on(_QWERTY);
```
Comment (drashna) [inline keyboards/xiudi/xd75/keymaps/raoeus/keymap.c:109]:
```suggestion
                layer_move(_QWERTY);
```

File: keyboards/xiudi/xd75/keymaps/raoeus/keymap.c [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            };
+            return false;
+            break;
+        case STENO:
+            if (record->event.pressed) {
+                layer_off(_LOWER);
+                layer_off(_RAISE);
+                layer_off(_BOTH);
+                layer_on(_STENO);
```
Comment (drashna) [inline keyboards/xiudi/xd75/keymaps/raoeus/keymap.c:118]:
```suggestion
                layer_move(_STENO);
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xiudi/xd75/keymaps/raoeus/keymap.c [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+     * | LShift | Z      | X      | C      | V      | B      | N      | M      | ,      | .      | /      | Enter  | 1      | 2      | 3      |
+     * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+     * | LCtl   | LAlt   | LGUI   | Steno  | Lower  | Space  | Space  | Raise  | Left   | Down   | Up     | Right  | 0      | .      | Enter  |
+     * '--------------------------------------------------------------------------------------------------------------------------------------'
+     */
+    [_QWERTY] = LAYOUT_ortho_5x15(KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_BSPC, KC_NUM, KC_PSLS, KC_PAST, KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_DEL, KC_P7, KC_P8, KC_P9, KC_ESC, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_P4, KC_P5, KC_P6, KC_LSFT, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_ENT, KC_P1, KC_P2, KC_P3, KC_LCTRL, KC_LALT, KC_LGUI, STENO, LOWER, KC_SPC, KC_SPC, RAISE, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_P0, KC_PDOT, KC_PENT),
```
Comment (fauxpark) [inline keyboards/xiudi/xd75/keymaps/raoeus/keymap.c:43]: Might be worth formatting this so it's not all on one line.

Comment (RAOEUS) [reply keyboards/xiudi/xd75/keymaps/raoeus/keymap.c]: Sure thing, that's no problem. I've reformatted so it's more human readable. I also tested building/flashing again to be sure all is good.
https://github.com/qmk/qmk_firmware/pull/17456/commits/a34c2d72390fcb5651e62a308593f2883778a889

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**